Src/Add-AzureDtlVM.ps1

```diff
@@ -220,14 +288,14 @@ try {
     [array] $failed = $vms | ? { $_.Properties.provisioningState -eq 'Failed' }
     LogOutput "Detected $($failed.Count) failed VMs"
 
-    RemoveBatchVms -vms $failed -batchSize $removeBatchSize -profilePath $profilePath -credentialsKind $credentialsKind
```

Comment:
Why removing profilePath? It seems that it should be used in the File branch inside the RemoveBatchVms function. It should be passed into Remove-AzureDtlLabVMs as a parameter.